scripts/variantstore/wdl/GvsImportGenomes.wdl

```diff
@@ -28,17 +34,27 @@ workflow GvsImportGenomes {
   Int num_samples = length(external_sample_names)
   Int max_auto_batch_size = 20000
 
+  # If they're a beta customer, figure out the depth of their scatter
+  Int beta_effective_load_data_batch_size = if num_samples < beta_customer_max_scatter then 1
+                                            else num_samples / beta_customer_max_scatter
+
   if ((num_samples > max_auto_batch_size) && !(defined(load_data_batch_size))) {
     call Utils.TerminateWorkflow as DieDueToTooManySamplesWithoutExplicitLoadDataBatchSize {
       input:
         message = "Importing " + num_samples + " samples but 'load_data_batch_size' not explicitly specified; limit for auto batch-sizing is " + max_auto_batch_size + " samples."
     }
   }
 
+
   # At least 1, per limits above not more than 20.
+  # But if it's a beta customer, use the number computed above
   Int effective_load_data_batch_size = if (defined(load_data_batch_size)) then select_first([load_data_batch_size])
-                                       else if num_samples < 1000 then 1
-                                            else num_samples / 1000
+                                       else if beta_customer_rate_limit then beta_effective_load_data_batch_size
+                                            else if num_samples < 1000 then 1
```

Comment:
That's a really good suggestion.  To be honest, I was frustrated with my inability to have an `if (beta_customer_rate_limit) {}` block that selectively overwrote the computed value of effective_load_data_batch_size, so this is the cleanest way I could think to do it with my limited WDL experience.  I was half expecting one of you to show me an easier way to handle the beta case

Reply:
I've restructured the code in that portion to unify the two cases a lot more, and made the variable name for beta behavior a little more self-explanatory.  I'm running it again (just in case some silly typo or something caused an issue), so I'll post the successful run tomorrow when I have it.